### PR TITLE
Jenkins deployment

### PR DESCRIPTION
**Pull request by paulschwarzenberger**
- Install Cloud Custodian across multiple TDR AWS accounts using Jenkins
- Deployment via Jenkinsfile
- Simple Email Service (SES) and Simple Queue Service (SQS) are in the TDR management account
- Use terraform templatefile function to simplify code by eliminating data template file resource

### CODE REVIEW

File: custodian/policies/ec2/ec2-unmark-encrypted.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      schedule: "rate(1 day)"
+      role: arn:aws:iam::{account_id}:role/CustodianUnmarkEncryptedEC2
+    resource: aws.ec2
+    description: |
+      Unmarks any EC2 instances scheduled for deletion if they are now using encrypted EBS
+    filters:
+      - type: ebs
+        key: Encrypted
+        value: True
+      - "tag:maid_status": not-null
```
Comment (TomJKing) [inline custodian/policies/ec2/ec2-unmark-encrypted.yml:14]: Should that be "maid"?

File: custodian/policies/ec2/ec2-mark-unencrypted.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        - RunInstances
+    resource: aws.ec2
+    description: |
+      Provisions a lambda and cloud watch event target that looks at all new instances
+      and marks those with unencrypted volumes for deletion
+    filters:
+      - type: ebs
+        key: Encrypted
+        value: false
+      - "tag:maid_status": absent
```
Comment (TomJKing) [inline custodian/policies/ec2/ec2-mark-unencrypted.yml:16]: Should that be "maid"?

**Review by TomJKing — COMMENTED** [review]
Looks OK. Have two minor comments.

File: custodian/policies/ec2/ec2-unmark-encrypted.yml (hunk 1 above)
Comment (paulschwarzenberger) [reply custodian/policies/ec2/ec2-unmark-encrypted.yml:14]: It's based on [this example](https://cloudcustodian.io/docs/aws/examples/ebsgarbagecollect.html)

**Review by TomJKing — APPROVED** [review]
Looks OK

**Comment by paulschwarzenberger** [comment]
maid_status is set as the default tag within the [source code](https://github.com/cloud-custodian/cloud-custodian/blob/master/c7n/tags.py)

**Pull request merged**